core/cibox-project-builder/files/drupal7/scripts/sniffers.yml

```diff
@@ -30,11 +30,12 @@
     build_reports_dir: build_reports
 
     composer_global_require:
-      - drupal/coder:dev-7.x-2.x
-      - squizlabs/php_codesniffer=1.5.6
+      - { package: 'drupal/coder', version: ':dev-8.x-2.x' }
+      - { package: 'squizlabs/php_codesniffer', version: '=2.*' }
+
     git_repos:
       - { branch: 'master', repo: 'http://git.drupal.org/sandbox/coltrane/1921926.git', name: 'DrupalSecure' } # git clone --branch master http://git.drupal.org/sandbox/coltrane/1921926.git DrupalSecure
-      - { branch: '7.x-1.x', repo: 'https://github.com/klausi/drupalpractice.git', name: 'DrupalPractice' } #git clone --branch 7.x-1.x https://github.com/klausi/drupalpractice.git DrupalPractice
+      - { branch: '8.x-1.x', repo: 'https://github.com/klausi/drupalpractice.git', name: 'DrupalPractice' } #git clone --branch 8.x-1.x https://github.com/klausi/drupalpractice.git DrupalPractice
```

Comment:
We can't set coder 8 for Drupal 7 scripts @Krasnyj 
Please use drupal8 folder which is already having Drupal 8 support